app/include/dt-bindings/zmk/led_indicators.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+/* Led Num Lock */
+#define LED_NUMLOCK (0)
```

Comment:
Any reason we're adding this and duplicating values already defined in `app/include/dt-bindings/zmk/hid_usage.h`

Reply:
These values are not exactly the same, they represent the index of the indicator within the bitmask. Having these values can be useful in the future if we want to add other indicators whose HID values are not consecutive.

Reply:
Ok, I would suggest:

1. Don't create duplicates.
2. If they're not needed in any DTS files (don't see why they would be, given our conversation about chosen nodes), then don't nest within `dt-bindings/`.
3. I might suggest using a naming convention like `ZMK_LED_NUMLOCK_BIT` for these.

I don't actually see these being *used* anywhere in the code. Am I missing something?

Reply:
These values should be used in the devicetree nodes, currently no boards/shields support LED indicators so you won't find these values in the code.

```
led_indicators {
    compatible = "zmk,led-indicators";
    led_numlock {
        ctrl = <&pwmleds>;
        index = <0>;
        binding = <LED_NUMLOCK>; 
    };
};
```

Since we have changed the devicetree definition, I will delete this file and move these definitions somewhere else.